py/client/build.gradle

```diff
@@ -86,17 +86,21 @@ tasks.getByName('check').dependsOn(Docker.registerDockerTask(project, 'testPyCli
         from('tests') {
             into 'project/tests'
         }
+        from ('requirements.txt') {
+            into 'project/'
+        }
     }
     containerDependencies.dependsOn = [deephavenDocker.healthyTask]
     containerDependencies.finalizedBy = deephavenDocker.endTask
     network = deephavenDocker.networkName.get()
     dockerfile {
         from('deephaven/python:local-build')
-        runCommand '''set -eux; \\
-                      pip3 install unittest-xml-reporting==3.0.4 pyarrow==7.0.0 protobuf==3.20.1 grpcio==1.46.0 bitstring==3.1.9 pandas==1.2.5 timeout-decorator==0.5.0;\\
-                      mkdir -p /out/report'''
         copyFile('project', '/project')
         workingDir('/project')
+        runCommand '''set -eux; \\
+                      mkdir -p /out/report; \\
+                      pip3 install --upgrade pip; \\
+                      pip3 install -r requirements.txt'''
```

Comment:
For readers of this code, would it make sense to change the name of this file, so it is clear that this is only for testing?

Reply:
good suggestion, added requirements-dev.txt